pyproject.toml

```diff
@@ -49,6 +49,8 @@ dependencies = [
     "scikit-image>=0.24.0,<0.25",
     "click<8.2.0",
     "seaborn>=0.13.2",
+    "jupyter>=1.1.1",
```

Comment:
These added dependencies are not related to this PR. I suggest that if there is a case for adding these, it should be done connected to a different issue and PR